packages/usa-prose/src/styles/_usa-prose.scss

```diff
@@ -9,4 +9,19 @@
     @include usa-list-styles;
     @include usa-table-styles;
   }
-}
+
+  > * + {
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      margin-top: 1.5em;
+    }
+
+    p {
+      margin-top: 1em;
+    }
+  }
```

Comment:
`* + &` gets confused when added to mixins, so moved these items directly inside the style rules.

Reply:
I came to a similar conclusion! I was able to get the same desired effect by placing these rules within the `& > {}` selector above it.

I `npm link`ed this to the [test repo](https://github.com/esheehan-gsl/uswds-layer-test) for #4962 and it build correctly 👍🏼